.env.local.example

```
@@ -0,0 +1,96 @@
+# The key to access the upgrade page, this is disabled on the default image
+# This needs to be a 16 character alphanumeric string
+UPGRADE_KEY=""
+# The key is used for various security related functions within MediaWiki
+# This needs to be a 64 character alphanumeric string
+SECRET_KEY=""
+
+
+# DB Configurations
+
+# The IP address or hostname of your database server
+DB_SERVER="local-wiki-mariadb"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)